packages/suite/package.json

```diff
@@ -26,10 +26,12 @@
         "dropbox": "5.1.0",
         "ethereumjs-tx": "^2.1.2",
         "ethereumjs-util": "^6.0.0",
+        "file-saver": "^2.0.2",
```

Comment:
Did not research,but is this package still needed? Can't be download triggered in anotherway, something like create a ref and click on it? Just brainstorming.

Reply:
Probably but this package works well (covers corner cases and has fallbacks).